pkg/postage/batchservice/batchservice_test.go

```diff
@@ -211,6 +233,41 @@ func TestBatchServiceTopUp(t *testing.T) {
 		if got.Value.Cmp(want) != 0 {
 			t.Fatalf("topped up amount: got %v, want %v", got.Value, want)
 		}
+		if testBatchListener.topupCount != 1 {
+			t.Fatalf("unexpected batch listener count, exp %d found %d", 1, testBatchListener.topupCount)
+		}
+	})
+
+	t.Run("passes without update", func(t *testing.T) {
```

Comment:
not sure what this means?

Reply:
This is to test that if the batch topup event is for a different owner, we should not see any topup events in the batchService.